src/tactic/suggest.lean

```diff
@@ -297,6 +309,37 @@ The default for `num` is `50`.
 For performance reasons `suggest` uses monadic lazy lists (`mllist`). This means that
 `suggest` might miss some results if `num` is not large enough. However, because
 `suggest` uses monadic lazy lists, smaller values of `num` run faster than larger values.
+---
```

Comment:
```suggestion

---
```
Lack of a line break was leading to weird formatting [here](https://robertylewis.com/mathlib_docs/tactics.html#suggest).